docgen/src/guide/Multi_index.md

```diff
@@ -6,22 +6,19 @@ category: guide
 navWeight: 50
 ---
 
-You can use multiple [`<InstantSearch>`](guide/<InstantSearch>.html) instances for cases like:
+Whenever you want to:
 
-* displaying hits from different indices
-* sharing a single SearchBox
-* any use case involving synchronizing widgets between different [`<InstantSearch>`](guide/<InstantSearch>.html) instances
+* display hits from different indices
+* share a single SearchBox
+* synchronize widgets between different indices
 
```

Comment:
I'm not sure yet. Apart from the synchronise that should be removed, we kind of agree that for now we should not highlight the fact that basically any widget can target any indices.